### ISSUE TITLE
Uncaught TypeError

### ISSUE THREAD

**Issue by tarlepp** [EasyCorp/easy-log-handler#15]
Just noticed that I'm getting following within my tests:

``` 
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to EasyCorp\EasyLog\EasyLogFormatter::formatThrowable() must be an instance of EasyCorp\EasyLog\Throwable, instance of Symfony\Component\Security\Core\Exception\AccessDeniedException given, called in /home/wunder/projects/symfony-flex-backend/vendor/easycorp/easy-log-handler/src/EasyLogFormatter.php on line 397 and defined in /home/wunder/projects/symfony-flex-backend/vendor/easycorp/easy-log-handler/src/EasyLogFormatter.php:405
Stack trace:
#0 /home/wunder/projects/symfony-flex-backend/vendor/easycorp/easy-log-handler/src/EasyLogFormatter.php(397): EasyCorp\EasyLog\EasyLogFormatter->formatThrowable(Object(Symfony\Component\Security\Core\Exception\AccessDeniedException))
#1 [internal function]: EasyCorp\EasyLog\EasyLogFormatter->EasyCorp\EasyLog\{closure}(Object(Symfony\Component\Security\Core\Exception\AccessDeniedException), 'exception')
#2 /home/wunder/projects/symfony-flex-backend/vendor/easycorp/easy-log-handler/src/EasyLogFormatter.php(400): array_walk_recursive(Arr in /home/wunder/projects/symfony-flex-backend/vendor/easycorp/easy-log-handler/src/EasyLogFormatter.php on line 405
``` 

Any ideas what is causing this?

**Comment by theofidry** [comment]
Likely https://github.com/EasyCorp/easy-log-handler/pull/14

**Comment by errogaht** [comment]
still not fixed JUST merge fix to master #16 or #17

**Comment by theofidry** [comment]
I understand that your frustration @errogaht but please keep in mind that the maintainers here have no obligation towards you and it is about free time, priorities and mood. Being aggressive will not help no matter how simple the task you see seem to be

**Comment by errogaht** [comment]
@theofidry, excuse me please, you are absolutely right!

**Comment by javiereguiluz** [comment]
Fixed by #14.
